tests/forms_tests/field_tests/test_jsonfield.py

```diff
@@ -0,0 +1,99 @@
+import uuid
+
+from tests.test_utils.json import CustomDecoder
```

Comment:
`test_utils` is not a place where you can put utils for tests, it is a module to test utils.

Reply:
Are you sure it's not [`utils_tests`](https://github.com/django/django/tree/master/tests/utils_tests)?

Reply:
Both directories contains tests. You can add decoders directly to `test_jsonfield.py` files.